base/gmp.jl

```diff
@@ -334,7 +334,7 @@ for (fJ, fC) in ((:-, :neg), (:~, :com))
     end
 end
 
-function <<(x::BigInt, c::Int32)
+function <<(x::BigInt, c::Int)
```

Comment:
Worth noting that `Int` is larger than `Culong` on win64, but the checked conversions in ccall should catch this (not so likely) case.